Packs/AWS-Enrichment-Remediation/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "AWS Enrichment and Remediation",
     "description": "Playbooks using multiple AWS content packs for enrichment and remediation purposes",
     "support": "xsoar",
-    "currentVersion": "1.1.17",
+    "currentVersion": "1.2.0",
```

```suggestion
    "currentVersion": "1.1.18",
```
Please don't forget to change the release notes file name as well

---

Updated the versions.